zulip/integrations/bridge_with_matrix/README.md

```diff
@@ -50,9 +66,29 @@ in a file called `matrix_bridge.conf`:
 
 * If you are running from the Zulip GitHub repo: run `python matrix_bridge.py -c matrix_bridge.conf`
 
-## Caveats for IRC mirroring
+
+## Notes regarding IRC
+
+### Usage for IRC bridges
+
+Matrix has been bridged to the listed
+[IRC networks](https://github.com/matrix-org/matrix-appservice-irc/wiki/Bridged-IRC-networks),
+where the 'Room alias format' refers to the `room_id` for the corresponding IRC channel.
+
+For example, for the freenode channel `#zulip-test`, the `room_id` would be
+`#freenode_#zulip-test:matrix.org`.
+
+Hence, this can also be used as a IRC <--> Zulip bridge.
```

Comment:
I think this punchline that the Matrix bridge can be used as an IRC <--> Zulip bridge needs to be the first sentence instead.

Reply:
Wouldn't that be confusing? After all, the IRC-bridge usage is just the usual Matrix-bridge usage, too...
BTW: For now, I removed the `tested connections` section in one of the comments in the code as I haven't tested the IRC->Matrix->Zulip path yet...

Reply:
Maybe instead say that Matrix can be used to *indirectly* bridge between IRC and Zulip. This sentence is the topic sentence of the entire paragraph.

Regarding the test cases, maybe they should be disabled instead of deleted?

Reply:
adapted :)
I'm going to adapt the old tests.